dataservice_publisher/app.py

```diff
@@ -44,6 +51,49 @@ async def authenticated(request: web.Request) -> bool:
     return False
 
 
+async def decide_content_type(request: web.Request) -> Optional[str]:
+    """Decide the content type of the response."""
+    logging.debug("Deciding content type")
+    if not request.headers.getone(hdrs.ACCEPT, None):
+        content_type = DEFAULT_CONTENT_TYPE  # pragma: no cover
+    elif "catalogs" in request.path:
+        accept_mime_types: List[str] = (
+            ",".join(request.headers.getall(hdrs.ACCEPT)).replace(" ", "").split(",")
+        )
+        for mime_type in accept_mime_types:
+            if mime_type in SUPPORTED_CONTENT_TYPES:
+                content_type = mime_type
+                break
+            elif mime_type == "*/*":
+                content_type = DEFAULT_CONTENT_TYPE
+                break
+            elif mime_type == "text/*":
+                content_type = DEFAULT_CONTENT_TYPE
+                break
```

Comment:
```suggestion
                break
            elif mime_type == "application/*":
                content_type = "application/ld+json"
                break
```
Fint å støtte `application/*` også, når vi støtter `text/*`